### PR TITLE
Add Runscope plugin

### PR DESCRIPTION
**Pull request by mansilladev**
Also a small update to ```app/enterprise/index.html``` (excess spaces making npm test fail)

### CODE REVIEW

**Comment by mansilladev** [comment]
Also small typo fix on word "Intelligence" (Galileo plugin)

**Comment by mansilladev** [comment]
@sinzone Please review when you get a chance.

File: app/plugins/runscope.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+## Installation
+
+Add the plugin to the list of available plugins on every Kong server in your cluster by editing the [kong.yml][configuration] configuration file:
+
+```yaml
+plugins_available:
+  - runscope
+```
+
+Every node in the Kong cluster must have the same `plugins_available` property value.
```
Comment (thibaultCha) [inline app/plugins/runscope.md]: This step/chapter is not required anymore for 0.6.0. Plugins included in Kong do not have to be added in the configuration anymore. The current documentation for other plugins reflect those changes.

File: app/plugins/runscope.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+It is important to be aware of performance when configuring this plugin. For example, be aware that logging the request and response bodies might slow down your traffic if your API is under heavy load. If your API works with significantly large request or response bodies, consider turning off this feature by updating the `log_body` configuration.
+
+## Configuration
+
+Configuring the plugin is straightforward, you can add it on top of an [API][api-object] (or [Consumer][consumer-object]) by executing the following request on your Kong server:
+
+```bash
+$ curl -X POST http://kong:8001/apis/{api}/plugins/ \
+    --data "name=runscope" \
+    --data "bucket_key=YOUR_BUCKET_KEY" \
```
Comment (thibaultCha) [inline app/plugins/runscope.md]: I think you mean `config.bucket_key`?

File: app/plugins/runscope.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    --data "config.access_token=YOUR_ACCESS_TOKEN"
+```
+
+`api`: The `id` or `name` of the API that this plugin configuration will target
+
+parameter                          | description
+---                                | ---
+`name`                             | The name of the plugin to use, in this case: `runscope`
+`config.access_token`              | The Runscope [access token][generate-access-token] (or personal access token) for the Runscope API.
+`config.bucket`                    | Your Runscope [bucket][runscope-buckets] ID where traffic data will be stored.
```
Comment (thibaultCha) [inline app/plugins/runscope.md]: `config.bucket_key` too?

File: app/plugins/runscope.md (hunk 1 above)
Comment (mansilladev) [inline app/plugins/runscope.md]: Got it. Addressed.

File: app/plugins/runscope.md (hunk 2 above)
Comment (mansilladev) [inline app/plugins/runscope.md]: Fixed.

File: app/plugins/runscope.md (hunk 3 above)
Comment (mansilladev) [inline app/plugins/runscope.md]: Fixed.

**Comment by thefosk** [comment]
@mansilladev we are ready to merge it, but you would need to fix the conflicts first

**Comment by thefosk** [comment]
@mansilladev never mind, I have manually applied the changes. Closing this.

**Pull request closed without merging**